beta/src/components/MDX/ExpandableExample.tsx

```diff
@@ -22,17 +22,29 @@ function ExpandableExample({
   excerpt,
   type,
 }: ExpandableExampleProps) {
+  const ref = React.useRef<null | HTMLDetailsElement>(null);
   const [isExpanded, setIsExpanded] = React.useState(false);
   const isDeepDive = type === 'DeepDive';
   const isExample = type === 'Example';
 
   return (
-    <div
+    <details
+      open={isExpanded}
+      onToggle={(e: any) => {
+        setIsExpanded(e.target!.open);
+      }}
+      onClick={(e) => {
```

Comment:
Should this be on the summary? Won't this mean that any links in the children, say, won't work?